config/locals.js

```diff
@@ -58,12 +58,16 @@ function loadCampaign(campaign) {
       app.locals.campaigns[campaign.id] = campaignDoc;
       logger.debug(`loaded app.locals.campaigns[${campaignDoc._id}]`);
 
-      if (campaignDoc.keyword) {
-        const keyword = campaignDoc.keyword.toUpperCase();
-        app.locals.keywords[keyword] = campaign.id;
-        logger.debug(`loaded app.locals.keyword[${keyword}]:${campaign.id}`);
+      if (campaignDoc.keywords) {
+        campaignDoc.keywords.map((campaignKeyword) => {
```

Comment:
You need `map` only when you're collection its results into an array. Simple `each` would suffice here.